lib/ProMotion/table/extensions/searchable.rb

```diff
@@ -57,6 +57,10 @@ def searchDisplayControllerWillBeginSearch(controller)
         self.table_view.setScrollEnabled false
         @table_search_display_controller.delegate.will_begin_search if @table_search_display_controller.delegate.respond_to? "will_begin_search"
       end
+
+      def searchDisplayController(controller, didLoadSearchResultsTableView: tableView)
+        tableView.rowHeight = self.table_view.rowHeight
+      end
```

Comment:
@markrickert Interestingly, @squidpunch had already submitted a pull request that should have fixed that earlier issue. Not sure what's going on here.